config/settings.yml

```diff
@@ -33,3 +34,6 @@ harvestdor:
 gdor:
   service_url: https://dorfetcher-url
   skip_heartbeat: true
+slack_notifications:
+  default_channel: '#spotlight-service-team'
+  api_token: 'your-api-token-here'
```

Comment:
🔑? Does this `api-token` need to point at at environment variable @jkeck?

Reply:
@camillevilla 🤔 no, I don't think so.  This can be set in a `development.local.yml` and I'll submit a PR to shared_configs to add our exhibits-bot API key if this is accepted.

Reply:
sul-dlss/shared_configs#429 & sul-dlss/shared_configs#430 👀